README.md

```diff
@@ -7,7 +7,7 @@ This repository contains the code we wrote during the *Udemy* edition of [Rock t
 - install Docker
 - either clone the repo or download as zip
 - open with IntelliJ as an SBT project
-- in a terminal window, navigate to the folder where you downloaded this repo and run `docker-compose up` to build and start the containers - we will need them to integrate various stuff e.g. Kafka with  Spark
+- in a terminal window, navigate to the folder where you downloaded this repo and run `docker compose up` to build and start the containers - we will need them to integrate various stuff e.g. Kafka with  Spark
```

Comment:
The command here is correct

Reply:
`docker-compose` is correct. 
But I believe we prefer `docker compose` today :-)
I've reverted it.